src/index.ts

```diff
@@ -41,7 +42,7 @@ export async function run(): Promise<void> {
 
     const creds: string = `${username}:${password}`;
 
-    await execute(podman, ['push', '--quiet', '--creds', creds, imageToPush, registryPath]);
+    await execute(podman, ['push', '--quiet', '--creds', creds, imageToPush, registryPath, `--tls-verify=${tlsVerify}`]);
```

Comment:
Can you edit this so that `--tls-verify` is only passed if `tlsVerify` is not-empty or null.